app/components/layouts/BackNavigation.tsx

```diff
@@ -6,22 +6,23 @@ import { useHistory } from "react-router-dom";
 // Renders a smart back link that handles pages visited directly or from referring page
 export const BackNavigation = ({
   defaultReturnTo,
+  content,
 }: {
-  defaultReturnTo: string;
+  defaultReturnTo?: string;
+  content: string;
 }) => {
   const history = useHistory();
 
   // The "POP" action is reserved as a default action for newly created history objects. `react-router-dom` depends on
   // another history library that defines this behavior. Read more:
   // https://github.com/remix-run/history/blob/main/docs/api-reference.md#reference
-  const backDestination =
-    history.action === "POP"
-      ? () => history.push(defaultReturnTo)
-      : () => history.goBack();
+  const backDestination = defaultReturnTo
+    ? () => history.push(defaultReturnTo)
+    : () => history.goBack();
 
   return (
     <Button onClick={backDestination} variant="linkWhite" arrowVariant="before">
-      Back to Service Listings
+      {content}
```

Comment:
What do you think about making this the implicit `{children}` passed in as a block to allow for more flexibility?